resources/views/layouts/app.blade.php

```diff
@@ -50,17 +53,20 @@
 
 {{-- <body class="font-sans antialiased biz-udpgothic-regular"> --}}
 
-<body class="biz-udpgothic-regular">
 
+<body class="biz-udpgothic-regular">
     <!-- Page Content -->
     <main>
         @yield('content')
     </main>
 
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"
-        integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous">
+    {{-- Bootstrap 5 JS --}}
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.8/dist/umd/popper.min.js"
```

Comment:
Please make this a local file. Avoid using cdn

Reply:
I have download and import bootstrap 5 into local. And, since I checked that navbar works properly with local-bs5 without popper, I removed CDN for it as well.